src/hotspot/cpu/riscv/templateTable_riscv_64.cpp

```diff
@@ -2900,7 +2885,7 @@ void TemplateTable::putfield_or_static(int byte_no, bool is_static, RewriteContr
   __ add(Rclass_or_obj, Rclass_or_obj, Roffset);
   __ fsd(F23_ftos, Rclass_or_obj, 0);
   if (!is_static && rc == may_rewrite) {
-    patch_bytecode(Bytecodes::_fast_dputfield, Rbc, Rscratch, true, byte_no);
+    //patch_bytecode(Bytecodes::_fast_dputfield, Rbc, Rscratch, true, byte_no); FIXME_RISCV
```

Comment:
I would also like this shortcut be verbose. So they will be visible during tests, so we could prioritize our work based on statistics of most triggered unimplemented parts